test/unit/fixtures.ts

```diff
@@ -213,54 +252,59 @@ export class Fixtures {
                     description: "O/D USAGE FEE",
                     transaction_id: "c1b3a5129487f96d03d22f66c2b57139",
                     transaction_type: "DEBIT",
+                    transaction_category: "DIRECT_DEBIT",
```

Comment:
same here `FEE_CHARGE`, since the `provider_transaction_category` is `CHARGE` and the description describes it as a fee